profiles/package.mask

```diff
@@ -92,11 +92,6 @@ app-misc/utimer
 # Removal on 2024-01-30.  Bug #849077.
 sys-block/noflushd
 
-# Michał Górny <mgorny@gentoo.org> (2023-12-31)
-# Buggy code that carries local patches already.  Last release in 2011.
-# Removal on 2024-01-30.  Bug #878629.
-net-proxy/tayga
-
 # Michał Górny <mgorny@gentoo.org> (2023-12-31)
```

Comment:
I'd like to see a separate commit for this

willing to maintain this package isn't really a good enough reason to convince us to let it stay in main portage tree.. the package get unmaintained will eventually got broken (buggy code, security vulnerability..), you probably need to think further, like fork-and-maintain it in upstream way? also this package lacks systemd unit file support

Reply:
> I'd like to see a separate commit for this

Done.

> the package get unmaintained will eventually got broken (buggy code, security vulnerability..)

If there are bugs or vulns I don't see why I couldn't fix them. The code is relatively simple.

I think the linked bug isn't a fair assessment of the upstream code considering it's a bug from a local patch, and that patch was adding a feature rather than fixing a bug.

> also this package lacks systemd unit file support

Is this urgent? I can add that but I don't run tayga directly.

Reply:
> > I'd like to see a separate commit for this
> 
> Done.
> 
> > the package get unmaintained will eventually got broken (buggy code, security vulnerability..)
> 
> If there are bugs or vulns I don't see why I couldn't fix them. The code is relatively simple.
> 
> I think the linked bug isn't a fair assessment of the upstream code considering it's a bug from a local patch, and that patch was adding a feature rather than fixing a bug.
> 

I'm not target any specific patch, but general suggestion for long term maintenance 

> > also this package lacks systemd unit file support
> 
> Is this urgent? I can add that but I don't run tayga directly.

not really urgent, you can improve it later